spark/src/test/scala/org/apache/mahout/sparkbindings/blas/BlasSuite.scala

```diff
@@ -39,11 +42,11 @@ class BlasSuite extends FunSuite with DistributedSparkSuite {
     val drmA = drmParallelize(m = inCoreA, numPartitions = 3)
     val drmB = drmParallelize(m = inCoreB, numPartitions = 2)
 
-    val op = new OpABt(drmA, drmB)
+    val op = OpABt(drmA, drmB)
 
     val drm = new CheckpointedDrmSpark(ABt.abt(op, srcA = drmA, srcB = drmB), op.nrow, op.ncol)
 
-    printf("AB' num partitions = %d.\n", drm.rdd.partitions.size)
+    printf("AB' num partitions = %d.\n", drm.rdd.partitions.length)
```

Comment:
in scala, we use size() to measure collection cardinality. using .length may cause conversion to a java collection.